lumen_runtime/src/time/datetime/web_sys.rs

```diff
@@ -0,0 +1,14 @@
+use js_sys::Date;
```

Comment:
You'll need an explicit, direct, declared dependency in the `Cargo.toml` for `lumen_runtime` to get this to work.